pluralize.js

```diff
@@ -330,9 +330,9 @@
     [/[^\u0000-\u007F]$/i, '$0'],
     [/([^aeiou]ese)$/i, '$1'],
     [/(ax|test)is$/i, '$1es'],
-    [/(alias|[^aou]us|tlas|gas|ris)$/i, '$1es'],
-    [/(e[mn]u)s?$/i, '$1s'],
     [/([^l]ias|[aeiou]las|[emjzr]as|[iu]am)$/i, '$1'],
+    [/(alias|christmas|[^aou]us|tlas|gas|ris)$/i, '$1es'],
```

Comment:
Moved this under the previous line because the previous line is covering `christmas` with an incorrect rule ... and it doesn't seem to cover any of the other regexes from this current line, so I wouldn't expect side effects.

Reply:
Have you checked with a word dictionary to ensure these rule order changes won't affect a bunch of untested words?